### PR TITLE
Add author meta

### PR DESCRIPTION
**Pull request by renshuki**
Hi,
Here is the fix for #96 
Thanks ;)

### CODE REVIEW

**Comment by benbalter** [comment]
LGTM. @pathawks?

**Comment by pathawks** [comment]
Have we found any documentation regarding how this information is used?

**Comment by renshuki** [comment]
Actually nothing in Facebook official documentation.  
- article:author   
- article:publisher  
must be an URL to a Facebook profile (or ID) to work properly.
  
In case those to tags are not defined, Facebook seems to use meta name="author".
So, even if you haven't a Facebook account you still can show up the author name of the article you share...

**Comment by renshuki** [comment]
Little "UP" on this PR before closing. No needs?

**Comment by pathawks** [comment]
I am :-1: without some documentation somewhere about how this works.

This is certainly a spa

**Comment by renshuki** [comment]
Thanks for the update!
As I explained above nothing written explicitly in Facebook documentation.

> article:author  

only seems supported officially by Facebook but it requires a Facebook profile which is not suitable for everyone use case.

Some topics are dealing with the subject:

_Source: Stack Overflow_

If you want some official documentation proof, wait and see.

**Comment by pathawks** [comment]
Fixes #96

**Comment by jhabdas** [comment]
FYI on use of `meta name=author` in combination with `meta name=article:author` and how Facebook treats each of them differently: https://github.com/jekyll/jekyll-seo-tag/issues/96#issuecomment-275840637

**Comment by jhabdas** [comment]
@renshuki I added info on how this works in my last comment. Think you could update the README and add some unit tests? If you need help just LMK.

File: lib/template.html [hunk 1]
```diff
@@ -108,6 +108,10 @@
   <meta property="og:title" content="{{ seo_page_title }}" />
 {% endif %}
 
+{% if seo_author %}
+  <meta name="author" content="{{ seo_author }}" />
+{% endif %}
```
Comment (jhabdas) [inline lib/template.html]: In the case `seo_author` is a key which maps to an item in `site.data.authors.yml` we should probably make the author name output using `site.data.authors[seo_author].name`.

Comment (renshuki) [inline lib/template.html]: Great idea! I made the change in the last commit.

**Comment by renshuki** [comment]
@jhabdas For unit tests, `site.data.authors` is actually in `context "twitter"` but the current feature is not related to Twitter so... Do you think a refactoring of the specs is preferable?

**Comment by jhabdas** [comment]
@renshuki sorry for the delayed reply. A refactor makes to me if it helps untangle namespaces / contexts. May want to hear from a project maintainer first though so you don't lose any time. I think @pathawks may be looking for Facebook to document the feature (assuming existing documentation cannot be found).

**Comment by renshuki** [comment]
@jhabdas Sure. I would greatly appreciate if Facebook could document this somewhere...

**Comment by jhabdas** [comment]
@renshuki IMHO it doesn't matter if they document it. The fact it works at all (as we've shown) and adds standards-based metadata enriching the page should be justification enough for the addition.

**Comment by pathawks** [comment]
> and adds standards-based metadata 

Where is that standard documented?

**Comment by jhabdas** [comment]
http://devdocs.io/html/element/meta

**Comment by jhabdas** [comment]
And here: https://www.w3.org/TR/html5/document-metadata.html#the-meta-element

**Comment by pathawks** [comment]
You're right 👍🏼

**Comment by benbalter** [comment]
Thanks @Thanks @renshuki @jhabdas! Merged via command line to confirm that rebasing allowed rubocop tests to pass.

**Comment by jhabdas** [comment]
w00t

**Comment by renshuki** [comment]
Thank you everyone 😄 
I will add some unit tests later for this if you want.

**Comment by benbalter** [comment]
> I will add some unit tests later for this if you want.

A yes. Even if just to confirm the field is there/filled out to prevent a regression.

**Comment by jhabdas** [comment]
@renahuki there lots of examples in the code you can learn from to get started. give it a shot! :)

**Pull request merged**